# Double: multiplier to apply to the maximum allowable vehicle deceleration limit so we plan under our capabilities
vehicle_decel_limit_multiplier : 0.50

# Double: The minimum distance in meters that the vehicle can be at before requiring a transition to the APPROACH state
min_approach_distance : 30.0

# Double: A buffer infront of the stopping location which will still be considered a valid stop. Units in meters
stopping_location_buffer : 5.0

# Double: A buffer in seconds around the green phase which will reduce the phase length such that vehicle still considers it non-green
green_light_time_buffer : 2.0

# Double: The minimum period in seconds which a maneuver plan must cover if the plugin wishes to control the whole plan
min_maneuver_planning_period : 15.1

# String: The name to use for this plugin during comminications with the arbitrator
strategic_plugin_name : WorkZonePlugin

# String: The name of the tactical plugin to use for Lane Following trajectory planning
lane_following_plugin_name : InLaneCruisingPlugin

# String: The name of the plugin to use for stop and wait trajectory planning
stop_and_wait_plugin_name : StopAndWaitPlugin

# String: The name of the plugin to use for intersection transit trajectory planning
intersection_transit_plugin_name : IntersectionTransitPlugin